File: datasets/IR-Plag/convertai-original-java-rest-kotlin/case-05/plagiarized/L6/08/Kasus5L6.kt
package IR

import java.util.Scanner

fun main(args: Array<String>) {
    // input data
    val inp = Scanner(System.`in`)
    print("Enter an integer: ")
    val angka = inp.next()
    for (i in angka.length - 1 downTo 0) {
        print(angka[i])
    }
    println()
}

File: datasets/IR-Plag/convertai-original-java-rest-typescript/case-05/plagiarized/L6/08/Kasus5L6.ts
import { Scanner } from 'readline-sync';

/**
 * 
 * @author B15130F5DDB6B5F1622EF91DAC4C1AAE
 */
function main(): void {
    //input data
    const inp: Scanner = new Scanner();
    console.log("Enter an integer: ");
    const angka: string = inp.next();
    for (let i: number = angka.length - 1; i >= 0; i--) {
        console.log(angka.charAt(i));
    }
    console.log();
}

main();

